File: michael-villarrubia-pantry-collab-BE/Models/PantryItem.cs
using System.Text.Json.Serialization;

namespace michael_villarrubia_pantry_collab_BE.Models
{
    public class PantryItem
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Image { get; set; } = string.Empty;
        public double Weight { get; set; }
        public int Calories { get; set; }
        public int QuantityInPantry { get; set; }
        public int PantryId { get; set; }
        [JsonIgnore]
        public Pantry Pantry { get; set; } = new Pantry();
    }
}

File: michael-villarrubia-pantry-collab-FE/src/app/Models/PantryItem.ts
import { Pantry } from './Pantry';

export class PantryItem {
  constructor(
    public id: number,
    public name: string,
    public image: string,
    public weight: number,
    public calories: number,
    public quantityInPantry: number,
    public pantryId: number
  ) {}
}
